### PR TITLE
Add v20.1-alpha release notes and update docs site for 20.1

### LINKED ISSUE #5877: v20.1.0-alpha.20191118 release notes
For https://github.com/cockroachdb/cockroach/issues/42368.

### PR DESCRIPTION
**Pull request by jseldess**
Fixes #5877.

### CODE REVIEW

File: releases/v20.1.0-alpha.20191118.md [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+### Docker image
+
+{% include copy-clipboard.html %}
+~~~shell
+$ docker pull cockroachdb/cockroach-unstable:v20.1.0-alpha.20191118
+~~~
+
+### General changes
+
+- Guidance on [how to contribute to CockroachDB](https://cockroachlabs.atlassian.net/wiki/x/QQFdB) have been moved to the public wiki at wiki.crdb.io. [#41542][#41542]
```
Comment (rmloveland) [inline releases/v20.1.0-alpha.20191118.md]: I think this wants to be singular? "has been moved"

Also wiki.crdb.io ought to be a clickable link

File: releases/v20.1.0-alpha.20191118.md [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+- Removed the `kv.bulk_io_write.addsstable_max_rate` [cluster setting](../20.1/cluster-settings.html). [#41745][#41745] {% comment %}doc{% endcomment %}
+- Improved the consistency checker's log output. [#41893][#41893]
+- When the replicas within a range are found to been corrupted, the outliers will be terminated. Previously, the leaseholder replica would terminate, regardless of which replicas disagreed with each other. This is expected to curb the spreading of corrupted data better than the previous approach. [#41902][#41902]
```
Comment (rmloveland) [inline releases/v20.1.0-alpha.20191118.md]: "are found to have been corrupted" or maybe "are found to be corrupted"?
"curb the spread of corrupted data"

File: releases/v20.1.0-alpha.20191118.md [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+- Fixed a bug where `SHOW ZONE CONFIGURATION` and `crdb_internal.zones` would show results for resources the user does not have permission to view. [#42066][#42066], [#42080][#42080]
+- Fixed a bug during planning for some queries that could cause an infinite loop and prevent the query from being cancelled. [#42082][#42082]
+- Fixed a bug that caused jobs for dropping tables to report an inaccurate status. [#42121][#42121]
+- Fixed a bug where rapid network disconnections could lead to cluster unavailability. [#41533][#41533]
+- Fixed a stack overflow that could occur with certain patterns of queries. [#41984][#41984]
+- Fixed some casts from `OID` to `TEXT`. [#41928][#41928]
+- Fixed a bug where some cluster setting changes were not reflected during currently running `IMPORT`s. [#42268][#42268]
+- Fixed bugs where `<date>:date` would result in the previous day (`<date-1>::date>`) when context local timestamp is set and the timezone is less than UTC+00:00, where `date_trunc()` for `TIMESTAMP` would produce incorrect results if a local timezone was set, and where `date_trunc()` for `DATE` would produce an incorrect negative timezone offset in a local timezone. [#42267][#42267]
```
Comment (rmloveland) [inline releases/v20.1.0-alpha.20191118.md]: This one was a bit hard to parse.  Maybe it would be easier to read as a list of semicolon-separated things, e.g.,

> Fixed bugs where: a thing happened; another thing happened; a third thing happened.

**Comment by jseldess** [comment]
Thanks, @rmloveland!

File: releases/v20.1.0-alpha.20191118.md (hunk 3 above)
Comment (jseldess) [reply releases/v20.1.0-alpha.20191118.md]: Done

File: releases/v20.1.0-alpha.20191118.md (hunk 2 above)
Comment (jseldess) [reply releases/v20.1.0-alpha.20191118.md]: Done.

File: releases/v20.1.0-alpha.20191118.md (hunk 1 above)
Comment (jseldess) [reply releases/v20.1.0-alpha.20191118.md]: Done.
